.pipelines/pipeline.android.yaml

```diff
@@ -0,0 +1,106 @@
+trigger:
+- master
+- push-refresh
+- v1-preview
+
+pr:
+- master
+- push-refresh
+- v1-preview
+
+pool:
+  name: Hosted Mac Internal
+  demands: java
+
+#Your build pipeline references an undefined variable named ‘getprop sys.boot_completed | tr -d '\r'’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab. See https://go.microsoft.com/fwlink/?linkid=865972
```

Comment:
Looks like something went amok here, are we confident that we don't actually need the line that it thinks is a variable name?

Reply:
removed this